pkg/kf/sources/util.go

```diff
@@ -53,3 +57,25 @@ type BuildTailerFunc func(ctx context.Context, out io.Writer, buildName, namespa
 func (f BuildTailerFunc) Tail(ctx context.Context, out io.Writer, buildName, namespace string) error {
 	return f(ctx, out, buildName, namespace)
 }
+
+// TektonLoggingShim implements BuildTailer for taskrun.LogReader
+// (https://godoc.org/github.com/tektoncd/cli/pkg/cmd/taskrun#LogReader).
+//
+// TODO: To keep the transition to Tekton small, we'll implement a function
+// here that implements the interface that knative-build expected. We should
+// obviously move away from this at some point.
+func TektonLoggingShim(ti tekton.Interface, ki kubernetes.Interface) BuildTailer {
+	return BuildTailerFunc(func(ctx context.Context, out io.Writer, buildName, namespace string) error {
+		reader := taskrun.LogReader{
+			Ns:       namespace,
+			Task:     buildName,
+			AllSteps: true,
+			Follow:   true,
+			Clients: &tektoncli.Clients{
+				Tekton: ti,
+				Kube:   ki,
+			},
+		}
+		panic(reader)
```

Comment:
Why is this panic here? Why create the LogReader just to panic?

Reply:
So I wasn't quite done and was only pushing to ensure I didn't lose work. It should be good now.